pentasharp/Controllers/BusSchedule.cs

```diff
@@ -264,9 +397,9 @@ public IActionResult GetReservations()
                         r.Schedule.ArrivalTime,
                         r.Schedule.Price,
                         r.Schedule.AvailableSeats,
-                        r.Schedule.Bus.BusNumber,
-                        r.Schedule.Route.FromLocation,
-                        r.Schedule.Route.ToLocation
+                        BusNumber = r.Schedule.Bus.BusNumber,
+                        FromLocation = r.Schedule.Route.FromLocation,
+                        ToLocation = r.Schedule.Route.ToLocation
```

Comment:
consistency, either similar approach to this  BusNumber = r.Schedule.Bus.BusNumber, in all parameters or the old approach r.Schedule.Bus.BusNumber,